jorge/js/app.test.js

```diff
@@ -0,0 +1,26 @@
+const {fireEvent, getByTestId, screen} = require('@testing-library/dom');
+require('@testing-library/jest-dom/extend-expect');
+const { JSDOM } = require('jsdom');
+const fs = require('fs');
+const path = require('path');
+
+const html = fs.readFileSync(path.resolve(__dirname, '../index.html'), 'utf8');
+
+let dom;
+let container;
+
+describe('index.html', () => {
```

Comment:
We are missing a test that checks if we get the div/p/span with the innerHTML of `Your balance is $0.00`